sys/arm64/conf/GENERIC-MORELLO-PURECAP-SYZKALLER-NODEBUG

```diff
@@ -0,0 +1,12 @@
+#
+# Pure capability ABI kernel for syzkaller fuzzing
+#
+# $FreeBSD$
+
+include "GENERIC-MORELLO-PURECAP-SYZKALLER"
+include "../../conf/std.nodebug"
+
+ident		GENERIC-MORELLO-PURECAP-SYZKALLER-NODEBUG
+
+options 	COVERAGE #re-enable coverage and kcov
```

Comment:
Move the comment before `options` as it relates to both COVERAGE and KCOV and add a space after `#`, i.e.
```
# Re-enable options after being disabled in std.nodebug
```